jck/compiler.lang/playlist.xml

```diff
@@ -350,6 +350,286 @@
 			<group>jck</group>
 		</groups>
 	</test>
+	<test>
+	<testCaseName>jck-compiler-lang-EXPR-group1</testCaseName>
+		<disables>
+			<disable>
+				<comment>Disabled on win and ppc64le due to backlog/issues/487. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
+				<platform>ppc64le_linux(_mixed)?|x86-64_windows(_mixed)?</platform>
```

Comment:
Please remove `(_mixed)?`. We do not need it anymore.

Reply:
Thanks. I've removed `(_mixed)?` from the playlists associated with this PR here. I'll have a separate PR to remove it from the rest of the tctk playlists.